### ISSUE TITLE
Confirm what you in telegramm group and you recive donate?

### ISSUE THREAD

**Issue by Hurd8x** [threealgos/Quantum-CRACKER#6]
Br

**Comment by threealgos** [comment]
yeas iam DRAGON GROVER.

**Comment by threealgos** [comment]
> Br

Yes I am Dragon Grover . by the way i am devloping Multi-Shors's Orcales quantum algorithm that i will hope that won't need a long time Preparations , i mean Orcales Preprations along all the circuit instead of Grover itterations.

**Comment by threealgos** [comment]
i mean Orcales Preprations along all the circuit instead of Grover itterations .

**Comment by Hurd8x** [comment]
> > Br
> 
> Yes I am Dragon Grover . by the way i am devloping Multi-Shors's Orcales quantum algorithm that i will hope that won't need a long time Preparations.

Ok
